game_api.txt

```diff
@@ -160,6 +160,28 @@ on_rotate(pos, node, user, mode, new_param2)
 ^ use on_rotate = screwdriver.disallow to always disallow rotation
 ^ use on_rotate = screwdriver.rotate_simple to allow only face rotation
 
+Sethome API
+--------------
+The Sethome API allows modders to use sethome's system in their own mods with the help of three functions :
+
+sethome.set_home(name)
```

Comment:
That function is designed to fit for [this precise line](https://github.com/minetest/minetest_game/pull/747/files#diff-76b4c3462c3911215a58c8916ea17235R74) of the chatcommand, although I agree that it would be good to have pos given to the function.